packages/web-components/fast-element/src/templating/repeat.ts

```diff
@@ -155,7 +155,7 @@ export class RepeatBehavior<TSource = any> implements Behavior, Subscriber {
      * @param args - The details about what was changed.
      */
     public handleChange(source: any, args: Splice[]): void {
-        if (source === this.itemsBinding) {
+        if (source === this.itemsBinding || source === this.items) {
```

Comment:
I don't think this is correct. When the `source` is equal to the array, that means that the array observer is reporting a list of splices/changes. If we check for this condition here, then that means that line 171 will never execute, which isn't correct.

Reply:
In what case would the source equal itemsBinding? Is there are case where source would be a Binding instead of an array?

Reply:
This handler is being used for multiple different observers at once. So, when the source is the itemsBinding, that means that the biding that returns the array has changed. When the source is the templateBinding, that means that the binding that returns the template has changed. And when the source is the items itself, that means that items have been added/remove/moved/etc. within the array but the array reference itself has not changed.